docs/core/knowledge-bases.rst

```diff
@@ -0,0 +1,496 @@
+:desc: Leverage information from knowledge bases inside conversations using ActionQueryKnowledgeBase
+       in open source bot framework Rasa.
+
+.. _knowledge_base_actions:
+
+Knowledge Base Actions
+======================
+
+.. edit-link::
+
+.. warning::
+   This feature is experimental.
+   We introduce experimental features to get feedback on implementations, so do please try it out.
+   However, the functionality might be changed or removed in the future.
+   If you have feedback (positive or negative) please comment in the `community forum <https://forum.rasa.com>`_.
```

Comment:
No, but we could create one. But maybe first release the feature, then create a forum post so that we can link to the documentation, and finally update the documentation to link to the correct forum post.

Reply:
Yes, I suppose that's probably the best way to do it.